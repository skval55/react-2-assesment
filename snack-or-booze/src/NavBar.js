import React from "react";
import "./NavBar.css";
import { NavLink } from "react-router-dom";
import { Navbar, Nav, NavItem } from "reactstrap";

function NavBar() {
  return (
    <div>
      <Navbar expand="md">
        <NavLink exact to="/" className="navbar-brand">
          Snack or Booze
        </NavLink>
        <div className="navItems">
          <Nav className="ml-auto" navbar>
            <NavItem>
              <NavLink to="/additem">Add Item</NavLink>
            </NavItem>
          </Nav>
          <Nav className="ml-auto" navbar>
            <NavItem>
              <NavLink to="/snacks">Snacks</NavLink>
            </NavItem>
          </Nav>
          <Nav className="ml-auto" navbar>
            <NavItem>
              <NavLink to="/drinks">drinks</NavLink>
            </NavItem>
          </Nav>
        </div>
      </Navbar>
    </div>
  );
}

export default NavBar;
