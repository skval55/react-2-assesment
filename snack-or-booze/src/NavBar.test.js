import { render, screen } from "@testing-library/react";
import NavBar from "./NavBar";
import { MemoryRouter } from "react-router-dom";
import React from "react";

// smoke test see if it renders
test("see if it renders", () => {
  render(
    <MemoryRouter>
      <NavBar />
    </MemoryRouter>
  );
});
test("see if it renders", () => {
  const { asFragment } = render(
    <MemoryRouter>
      <NavBar />
    </MemoryRouter>
  );
  expect(asFragment()).toMatchSnapshot();
});
