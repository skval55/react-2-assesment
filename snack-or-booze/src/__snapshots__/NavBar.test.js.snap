// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`see if it renders 1`] = `
<DocumentFragment>
  <div>
    <nav
      class="navbar navbar-expand-md"
    >
      <a
        aria-current="page"
        class="navbar-brand active"
        href="/"
      >
        Snack or Booze
      </a>
      <div
        class="navItems"
      >
        <ul
          class="ml-auto navbar-nav"
        >
          <li
            class="nav-item"
          >
            <a
              href="/additem"
            >
              Add Item
            </a>
          </li>
        </ul>
        <ul
          class="ml-auto navbar-nav"
        >
          <li
            class="nav-item"
          >
            <a
              href="/snacks"
            >
              Snacks
            </a>
          </li>
        </ul>
        <ul
          class="ml-auto navbar-nav"
        >
          <li
            class="nav-item"
          >
            <a
              href="/drinks"
            >
              drinks
            </a>
          </li>
        </ul>
      </div>
    </nav>
  </div>
</DocumentFragment>
`;
